install/README.md

```diff
@@ -48,6 +48,9 @@ Now you're ready to deploy an app:
 
 - View the available [sample apps](../serving/samples) and deploy one of your
   choosing.
+  
+- Walk through a Google codelab: 
```

Comment:
Since this list is introduced with a colon, it looks a little funny to have another colon here. Can we do something like "Walk through the Google codelab, [Link]."?

Reply:
done